.evergreen/config.yml

```diff
@@ -2431,6 +2431,20 @@ buildvariants:
   display_name: "PyOpenSSL ${platform} ${python-version-windows} ${auth}"
   tasks:
     - '.replica_set'
+  rules:
```

Comment:
Is this actually a problem when using PyOpenSSL, or is it only a problem with CPython's ssl module?

Reply:
Oh right, I was confused by the other test failure. PyOpenSSL+3.10+MongoDB 2.6 works fine.